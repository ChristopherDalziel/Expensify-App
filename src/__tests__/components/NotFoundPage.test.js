import React from "react";
import { shallow } from "enzyme";
import NotFoundPage from "../../components/pages/NotFoundPage";

import toJSON from "enzyme-to-json";

test("Should render page not found page correctly", () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
