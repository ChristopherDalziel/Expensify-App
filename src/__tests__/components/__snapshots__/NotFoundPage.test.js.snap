// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render page not found page correctly 1`] = `ShallowWrapper {}`;
